shared/teams/team/settings-tab/container.tsx

```diff
@@ -60,6 +64,7 @@ export default Container.connect(
   (stateProps, dispatchProps) => {
     return {
       ...stateProps,
+      loadWelcomeMessage: dispatchProps._loadWelcomeMessage,
```

Comment:
we typically prefix functions that won't be directly passed through with `_` so since this isn't the case i'd just call it `loadWelcomeMessage` in the mapDispatch call above